src/devices/DimmableLightingDevice/DimmableLightingDevice.ts

```diff
@@ -136,24 +81,224 @@ export default class DimmableLightingDevice extends InsteonDevice {
 	public switch(state: boolean, level?: Byte, fast: boolean = false){
 
 		return state ? fast ? this.LightOnFast(level)  : this.LightOn(level)
-					       : fast ? this.LightOffFast(level) : this.LightOff(level);
+					       : fast ? this.LightOffFast() : this.LightOff();
 
 	}
 
-	public async getDeviceStatus(){
+    public async getDeviceStatus(){
 
 		// Getting status
 		const statusPacket = await this.statusRequest();
 
 		// Parsing status out
 		return {
+			// state: statusPacket.cmd === 0x00 ? "off" : "on",
```

Comment:
Was contemplating adding the state `on|off` in addition to the level.

Reply:
I think just the level would be fine.  The user can do the calculation on their own